x-pack/test/functional/apps/ml/short_tests/model_management/model_list.ts

```diff
@@ -157,6 +181,59 @@ export default function ({ getService }: FtrProviderContext) {
         await ml.securityUI.logout();
       });
 
+      describe('Add trained model flyout', () => {
+        before(async () => {
+          await ml.testExecution.logTestStep('Open the Add Trained Model Flyout');
+          await ml.trainedModelsFlyout.open();
+          await ml.testExecution.logTestStep('Assert the Add Trained Model Flyout is open');
+          await ml.trainedModelsFlyout.assertOpen();
+        });
+
+        after(async () => {
+          await ml.testExecution.logTestStep('Close the Add Trained Model flyout');
+          await ml.trainedModelsFlyout.close();
+          await ml.testExecution.logTestStep('Assert the Add Trained Model flyout is closed');
+          await ml.trainedModelsFlyout.assertClosed();
+        });
+
+        it('should contain a Click to Download and a Manual Download tab', async () => {
+          await ml.testExecution.logTestStep(
+            'Assert the "Click to Download" and "Manual Download" tabs exists'
+          );
+          await ml.trainedModelsFlyout.assertFlyoutTabs(['Click to Download', 'Manual Download']);
```

Comment:
should be safer by the data test subj 
```suggestion
          await ml.trainedModelsFlyout.assertFlyoutTabs(['clickToDownload', 'manualDownload']);
```